src/aggregator-cli/GitHubVersionResponse.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.TeamFoundation.TestManagement.WebApi.Legacy;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace aggregator.cli
+{
+    class GitHubVersionResponse
+    {
+        public const string CacheFileName = "githubresponse.cache";
```

Comment:
Instead of the current directory, it is better to use the same directory where logon data is cached, which is certanly writeable
`public string CacheFileName => Path.Combine( Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData, Environment.SpecialFolderOption.Create), "aggregator-cli", "githubresponse.cache");`